object_detection/scripts/object_detection_node.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+
+import rospy
+
+# Import msg types
+from darknet_ros_msgs.msg import BoundingBoxes, BoundingBox
+from sensor_msgs.msg import Image
+from vortex_msgs.msg import BBox, BBoxes
+from std_msgs.msg import Int64, Float64
+from geometry_msgs.msg import PointStamped, Point
+import tf
+
+# Import classes
+from size_estimator import SizeEstimatorClass as SEC
+from coord_pos import CoordPositionClass as CPC
+
+class OD_NODE():
+    test = 'T'
+    def __init__(self):
+        rospy.init_node('object_detection_node')
+        self.estimatorSub = rospy.Subscriber('/darknet_ros/bounding_boxes', BoundingBoxes, self.estimatorSub_callback)
+        # self.imageSub = rospy.Subscriber('/darknet_ros/detection_image', Image, self.img_CB)
+        self.estimatorPub = rospy.Publisher('/object_detection/size_estimates', BBoxes, queue_size= 1)
+        # self.pointPub = rospy.Publisher('/object_detection/object_point', PointStamped, queue_size= 1)
+        self.size_estimator = SEC()
+        self.coord_positioner = CPC()
+
+    def estimatorSub_callback(self, data):
+        # Allocates msg data to local variables
+        # In order to process abs size
```

Comment:
It's good that you added documentation here, but to keep things uniform, you should have a look at [this styleguide](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) and format the docstring accordingly.